specifyweb/frontend/js_src/lib/components/Atoms/className.ts

```diff
@@ -12,9 +12,9 @@ export const dialogIconTriggers = {
 };
 
 const buttonClassName = 'button';
-const secondaryButton = `${buttonClassName} hover:brightness-90 dark:hover:brightness-150 bg-[color:var(--secondary-button-color)] text-gray-800 shadow-sm
+const secondaryButton = `${buttonClassName} hover:brightness-90 dark:hover:brightness-125 bg-[color:var(--secondary-button-color)] text-gray-800 shadow-sm
```

Comment:
make sure to do quick testing of at least a few places where secondary and secondary light button is used to make sure the hover effect is visible enough even after this change